README.md

```diff
@@ -80,6 +80,24 @@ shpwrite.download({
 Given a [GeoJSON](http://geojson.org/) FeatureCollection as an object,
 converts convertible features into Shapefiles and triggers a download.
 
+### `downloadMany(geojsonArray)`
+
+Given array of a [GeoJSON](http://geojson.org/) FeatureCollections,
+converts convertible features into Shapefiles and triggers a download.
+Supports only polygons.
+
+```js
+var options = {
+    file: 'myshapes'
+    folder: 'myshapes',
+    prj: 'wkt string' (required),
+    layerNames: [names_of_featureCollections] // same order that in featureCollections. these names are generated to shapefilenames
```

Comment:
pass layer names as keys along with featureCollection `{ <layerName1>: <featureCollection1>, <layerName2>: <featureCollection2> }` -> separate options.layerNames not needed.